src/signals/incident-management/containers/StandardTextsAdmin/components/OverviewPage/OverviewPage.tsx

```diff
@@ -25,11 +25,30 @@ export const OverviewPage = () => {
 
   const { get, data, isLoading, error } = useFetch<StandardTextsData>()
 
+  const fetchData = useCallback(
+    (value?: string) => {
+      const params = value ? `?q=${value}` : ''
```

Comment:
if its empty, it will just return the full list?

Reply:
jep!